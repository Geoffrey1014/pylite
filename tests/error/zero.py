print 'Start'
0/0
print 'End'
