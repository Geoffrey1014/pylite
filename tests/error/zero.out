Start
ZeroDivisionError:
  At zero.py, line 2, in __init__()
